### PR TITLE
LPS-83811 replace bindKey from dynamic-data-maping

### PR DESCRIPTION
**Pull request by iiventura**

### CODE REVIEW

**Comment by iiventura** [comment]
ci:test

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/edit_template_form.jspf [hunk 1]
```diff
@@ -55,8 +55,7 @@ String fieldsJSONArrayString = fieldsJSONArray.toString();
 	);
 
 	function <portlet:namespace />setFieldsHiddenAttributes(formBuilder, mode, item, index) {
-		var hiddenAttributesMap = formBuilder.MAP_HIDDEN_FIELD_ATTRS;
```
Comment (carloslancha) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/edit_template_form.jspf]: Why removing this?

File: modules/apps/frontend-js/frontend-js-web/test/liferay/util/bindKey.es.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            'should invoke the method of the object[key]',
+            () => {
+                
+                expect(bound('!')).toEqual('hi fred!');
+                  
+            }
+        );
+
+        it(
+            'should modified the method of the object',
```
Comment (carloslancha) [inline modules/apps/frontend-js/frontend-js-web/test/liferay/util/bindKey.es.js]: I'd reword this to "should maintain the param binding even if the method changes"

File: modules/apps/frontend-js/frontend-js-web/test/liferay/util/bindKey.es.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            'user': 'fred',
+            'greet': function(greeting, punctuation) {
+              return greeting + ' ' + this.user + punctuation;
+            }
+          }; 
+
+          var bound = bindKey(object, 'greet', 'hi');
+
+      it(
+            'should invoke the method of the object[key]',
```
Comment (carloslancha) [inline modules/apps/frontend-js/frontend-js-web/test/liferay/util/bindKey.es.js]: I'd reword this to "should invoke the object method with binded params"
